diff --git a/node_modules/cypress/lib/exec/spawn.js b/node_modules/cypress/lib/exec/spawn.js
index ed13727..37678d8 100644
--- a/node_modules/cypress/lib/exec/spawn.js
+++ b/node_modules/cypress/lib/exec/spawn.js
@@ -138,6 +138,8 @@ module.exports = {
         child.on('close', resolve);
         child.on('error', reject);
 
+        console.log('process.stdin is TTY?', process.stdin.isTTY)
+        console.log('child.stdin is TTY?', child.stdin.isTTY)
         child.stdin && child.stdin.pipe(process.stdin);
         child.stdout && child.stdout.pipe(process.stdout);
 
