diff --git a/node_modules/cypress/lib/exec/spawn.js b/node_modules/cypress/lib/exec/spawn.js
index ed13727..537e962 100644
--- a/node_modules/cypress/lib/exec/spawn.js
+++ b/node_modules/cypress/lib/exec/spawn.js
@@ -138,7 +138,7 @@ module.exports = {
         child.on('close', resolve);
         child.on('error', reject);
 
-        child.stdin && child.stdin.pipe(process.stdin);
+        // child.stdin && child.stdin.pipe(process.stdin);
         child.stdout && child.stdout.pipe(process.stdout);
 
         // if this is defined then we are manually piping for linux
