diff --git a/node_modules/cypress/lib/exec/spawn.js b/node_modules/cypress/lib/exec/spawn.js
index ed13727..19c1fae 100644
--- a/node_modules/cypress/lib/exec/spawn.js
+++ b/node_modules/cypress/lib/exec/spawn.js
@@ -42,7 +42,7 @@ function needsEverythingPipedDirectly() {
 
 function getStdio(needsXvfb) {
   if (needsEverythingPipedDirectly()) {
-    return 'pipe';
+    return 'inherit';
   }
 
   // https://github.com/cypress-io/cypress/issues/921
